.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -10,8 +10,5 @@ changes._
 
 ## Acceptance checklist
 
-- [ ] I have evaluated the
-      [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines)
-      for this PR.
-- [ ] I have tested this PR in
-      [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
+- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
+- [ ] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
```

Comment:
what's this formatting about?

Reply:
@peripheralist just removing the newlines, think VSCode auto-formatting adds them
<img width="310" alt="Screen Shot 2022-03-24 at 8 40 07 pm" src="https://user-images.githubusercontent.com/94939382/159893388-2dfcccbb-f5ec-4cd8-a61b-8e1426576122.png">